Ghidra/Features/Base/src/main/java/ghidra/plugin/importer/ImporterUtilities.java

```diff
@@ -100,6 +100,10 @@ public static void setProgramProperties(Program program, FSRL fsrl, TaskMonitor
 				fsrl.getMD5() != null) {
 				program.setExecutableMD5(fsrl.getMD5());
 			}
+			if ((program.getExecutableSHA256() == null || program.getExecutableSHA256().isEmpty()) &&
+				fsrl.getSHA256() != null) {
+				program.setExecutableMD5(fsrl.getSHA256());
```

Comment:
This and the line above cause a compilation error (getSHA256 does not exist in FSRL). Also, I think you meant to call setExecutableSHA56, not MD5. I'm actually not certain why we try to populate the MD5 in `ImporterUtilties` at all.... That said, I don't think you need to populate the SHA256 here, either. I think your changes to `AbstractProgramLoader` would be sufficient, even when loaded from a special file system. I recommend deleting this change.

Reply:
Done (in b799186f980f1374914218f239e4fb81e7bfc941). That's precisely why I labelled it "draft" :)